build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
These are new functions, so there is no backward compatibility issue.

On the other side I'm not for using an extra implicit conversion for "magic".